footer.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">    
+        <link rel="stylesheet-footer" href="CSS/footer.css">
+    </head>
+
+    <body>
+      <div class="mobile-footer">
+
+        <div class="h1-footer">
+        <h1>Cakes</h1>
+        </div>    
+
+        <div class="nav-footer">
+
+          <div class="links-footer">
+            <a href="#our-cakes">Наши торты</a>
+            <a href="#personal-order">Индивидуальный заказ</a>
+            <a href="#reviews">Отзывы</a>
+            <a href="#about-us">О нас</a>
+            <a href="#our-advantages">Наши преимущества</a>
+            <a href="#contacts">Контакты</a>
+          </div>
+        
+        </div>
+
+        <div class="pnumber-footer">
+            <p>+7 906 899 92 96</p>
```

Comment:
It's better to use anchor telephone tag in this situation so users can click on this phone number and native phone application will open. You can read about it here:
https://www.bounteous.com/insights/2014/01/16/make-phone-numbers-clickable-and-trackable-across-mobile-devices-javascript/